libs/ui-lib/lib/ocm/components/clusterConfiguration/networkConfiguration/VirtualIPControlGroup.tsx

```diff
@@ -137,7 +150,7 @@ export const VirtualIPControlGroup = ({
             }
             name="apiVip"
```

Comment:
Should't we refer to the new field too? The name is used to access the field in the cluster.
```
 if (vipDhcpAllocation && cluster[vipName]) 
```